### PR TITLE
Use psammead-brand

### PR DESCRIPTION
**Pull request by radiocontrolled**
Resolves https://github.com/BBC-News/simorgh/issues/1001

Remove Brand component and replace it with psammead-brand

Header
screen shot 2018-12-14 at 10 01 14

Footer
screen shot 2018-12-14 at 10 01 21

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by radiocontrolled** [comment]
I believe I should pause on completing this until https://github.com/BBC-News/psammead/pull/202 is merged into psammead. 

Psammead 202 is causing an error in snapshots, which is affecting this PR.

**Comment by radiocontrolled** [comment]
I have blocked this on https://github.com/BBC-News/psammead/pull/202

File: package.json [hunk 1]
```diff
@@ -58,11 +58,12 @@
   },
   "homepage": "https://github.com/bbc-news/simorgh#readme",
   "dependencies": {
-    "@bbc/psammead-copyright": "^0.2.0",
+    "@bbc/psammead-brand": "^0.1.1",
     "@bbc/psammead-headings": "^0.1.2",
+    "@bbc/psammead-paragraph": "^0.1.3",
+    "@bbc/psammead-copyright": "^0.2.0",
```
Comment (ChrisBAshton) [inline package.json]: Can you make this list alphabetical? Thanks!

**Review by ChrisBAshton — COMMENTED** [review]
One quick refactor on the package.json while we're here, then we're good to go 👍

File: package.json (hunk 1 above)
Comment (radiocontrolled) [reply package.json]: @ChrisBAshton I have addressed this comment

**Review by bcmn — APPROVED** [review]

**Review by sareh — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM.

**Pull request merged**